Docusaurus Canary - Can't resolve useThemeContext

Running against Docusaurus canary (which has some fixes for docs sidebar issues - redocusaurus is throwing an error: 

```
Module not found: Error: Can't resolve '@theme/hooks/useThemeContext' in 'X:\Development\Repositories\CIPP Project\website\node_modules\docusaurus-theme-redoc\dist-jsx\theme\Redoc'
```

It appears similar to #96 and looks to be caused by this breaking change: https://github.com/facebook/docusaurus/pull/6289

--- a/packages/docusaurus-theme-redoc/src/theme/ApiSchema/ApiSchema.tsx
+++ b/packages/docusaurus-theme-redoc/src/theme/ApiSchema/ApiSchema.tsx
@@ -3,7 +3,7 @@ import {
   usePluginData,
   useAllPluginInstancesData,
 } from '@docusaurus/useGlobalData';
-import useThemeContext from '@theme/hooks/useThemeContext';
+import { useColorMode } from '@docusaurus/theme-common';
 import clsx from 'clsx';
 import { ThemeProvider } from 'styled-components';
 import { SchemaDefinition, AppStore } from 'redoc';
@@ -17,7 +17,7 @@ const ApiSchema: React.FC<Props> = ({
   pointer,
   ...rest
 }: Props): JSX.Element => {
-  const { isDarkTheme } = useThemeContext();
+  const { isDarkTheme } = useColorMode();
   const allData = useAllPluginInstancesData<Spec>('docusaurus-plugin-redoc');
   const { lightTheme, darkTheme, redocOptions } = usePluginData<GlobalData>(
     'docusaurus-theme-redoc',

--- a/packages/docusaurus-theme-redoc/src/theme/Redoc/Redoc.tsx
+++ b/packages/docusaurus-theme-redoc/src/theme/Redoc/Redoc.tsx
@@ -1,6 +1,6 @@
 import React, { useMemo } from 'react';
 import { usePluginData } from '@docusaurus/useGlobalData';
-import useThemeContext from '@theme/hooks/useThemeContext';
+import { useColorMode } from '@docusaurus/theme-common';
 import { Redoc as RedocComponent, RedocStandalone, AppStore } from 'redoc';
 import { RedocProps as Props, GlobalData } from '../../types/common';
 import './styles.css';
@@ -13,7 +13,7 @@ import './styles.css';
  * Released under the MIT License
  */
 function Redoc(props: Props): JSX.Element {
-  const { isDarkTheme } = useThemeContext();
+  const { isDarkTheme } = useColorMode();
   const { lightTheme, darkTheme, redocOptions } = usePluginData<GlobalData>(
     'docusaurus-theme-redoc',
   );

